src/util/battery/battery.cpp

```diff
@@ -16,16 +16,16 @@
 #include "util/math.h"
 
 // interval (in ms) of the timer which calls update()
-constexpr int kiUpdateInterval = 5000;
+constexpr int kBatteryUpdateIntervalMS = 5000;
```

Comment:
```suggestion
constexpr int kBatteryUpdateIntervalMillis = 5000;
```
That is the usual suffix in the codebase.

Reply:
Why not using the proper type `std::chrono::milliseconds` instead of adding this information to the name?

Reply:
Thank you, much better solution